kiwi-json/src/commonTest/kotlin/org/fromsource/kiwi/json/ExtensionTest.kt

```diff
@@ -11,7 +11,7 @@ class ExtensionTest {
                 .split('a', 0)
                 .should
                 .haveFirstEqualTo("This is a sentence".toCharList())
-                .haveSecondBeEqual("".toCharList())
+                .haveSecondEqualTo("".toCharList())
     }
 
```

Comment:
should we add tests for Triple here as well?

Reply:
No, this test is for checking `split` function used for json